content/_data/adopters/Anchore.adoc

```diff
@@ -1,6 +1,6 @@
 ---
 name: "Anchore"
 url: https://anchore.com/
-detailsUrl: https://jenkinsistheway.io/user-story/to-understand-and-simplify-your-software-lifecycle/
```

Comment:
was this one just never finished or something? I don't have it nor anchor in my export.

Reply:
I think it might have been taken down from memory

Reply:
I assume it was taken down.  I think we can still call them an "adopter" because they have a blog post that describes Jenkins in their context.